app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
+
+class PollTwitter extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'suggestive:poll-twitter';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Polls Twitter for any tweets with the configured hashtag.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $users = User::where('role', 'owner')
+                     ->whereNotNull('hashtag')
+                     ->get(['id', 'name', 'hashtag', 'last_tweet_id']);
+
+        if ($users) {
+            Log::info('Found owners to poll Twitter for', ['count' => $users->count()]);
+
+            $twitter = app('TwitterClient');
```

Comment:
Yeah, for some reason I had trouble getting this to work in another project use used `TwitterOAuth` with, but that's because I was passing the additional parameters to it, so needed to use `app()->make()` to pass the extra parameters in.

I'll push an update shortly that fixes this and pushes the functionality off to a `Job` :+1: